packages/contracts/docs/core/01-how-it-works/01-core/01-dao/index.md

```diff
@@ -0,0 +1,73 @@
+---
+title: The DAO Contract
+---
+
+## The Identity and Basis of Your Organization
+
+In this section, you will learn about the core functionality of every aragonOS DAO.
+
+The `DAO` contract is the identity and basis of your organization. It is the address carrying the DAO’s ENS name, metadata, and holding the funds. Furthermore, it has **six base functionalities** being commonly found in other DAO frameworks in the ecosystem.
+
+### 1. Execution of Arbitrary Actions
+
+The most important and basic functionality of your DAO is the **execution of arbitrary actions**, which allows you to interact with the rest of the world, i.e., calling methods in other contracts services sending assets to other addresses.
+In our framework, actions are represented by a solidity struct:
+
+```solidity title="contracts/core/IDAO.sol"
+/// @notice The action struct to be consumed by the DAO's `execute` function resulting in an external call.
+/// @param to The address to call.
+/// @param value The native token value to be sent with the call.
+/// @param data The bytes-encoded function selector and calldata for the call.
+struct Action {
+  address to;
+  uint256 value;
+  bytes data;
+}
+```
+
+Actions can be calls to external contracts, plugins, or the aragonOS DAO framework infrastructure, for example, to [setup a plugin](../../02-framework/02-plugin-repository/04-plugin-setup.md).
+Actions are typically scheduled in a proposal in a governance [plugin customizing your DAO](../03-plugins/index.md).
+
+Multiple `Action` structs can be put into one `Action[]` array and executed in a single transaction via the `execute` function.
+To learn more about actions and advanced features of the DAO executor, visit the [A Deep Dive Into Actions](./01-actions.md) section.
+
+### 2. Asset Management
+
+The DAO provides basic **asset management** functionality to deposit, withdraw, and keep track of
+
+- [ERC-20 (Token Standard)](https://eips.ethereum.org/EIPS/eip-20),
+- [ERC-721 (NFT Standard)](https://eips.ethereum.org/EIPS/eip-721), and
+- [ERC-1155 (Multi Token Standard)](https://eips.ethereum.org/EIPS/eip-1155)
```

Comment:
This might be a bit of an over promise, given that transfers are encoded via SDK and deposits are currently only handled for ETH/ERC20. The rest of cases, simply happen without any intervention

Reply:
What do you suggest we should do?

Reply:
I would either not talk about this or explain that token management is done via `execute` by encoding the calldata (withdraw), and that for  `deposit` we have a bit of a mixed situation

Reply:
We implemented the `onERC721Received`, `onERC1155Received`, and `onERC1155BatchReceived` callbacks so we actively did work to support these tokens.

I guess the sentence is still fine although there is no explicit `deposit` and `withdraw` function.

Reply:
But the deposit action can still happen without the SDK and in the protocol itself (which is what this documentation is for) you can actually deposit these token standards, right?

Potentially the over-promise is only the tracking itself?